test/datadog/baseline/agent-clusterchecks-deployment_default.yaml

```diff
@@ -33,144 +33,138 @@ spec:
         app.kubernetes.io/component: clusterchecks-agent
         admission.datadoghq.com/enabled: "false"
         app: datadog-clusterchecks
-        
+
       name: datadog-clusterchecks
       annotations:
         checksum/clusteragent_token: ce75393cbdc42f29bc23068e7ebd685d85a9d00f6eab86c9030153d065d7c2bc
         checksum/install_info: 5dc2ee139450be3da942ecdcd059c0481d4422714c642d6592cba7c2779ee0f4
     spec:
       serviceAccountName: datadog-cluster-checks
       automountServiceAccountToken: true
-      imagePullSecrets:
-        []
+      imagePullSecrets: []
       initContainers:
-      - name: init-volume
-        image: "gcr.io/datadoghq/agent:7.63.0"
-        imagePullPolicy: IfNotPresent
-        command: ["bash", "-c"]
-        args:
-          - cp -r /etc/datadog-agent /opt
-        volumeMounts:
-          - name: config
-            mountPath: /opt/datadog-agent
-            readOnly: false # Need RW for writing agent config files
-        resources:
-          {}
-      - name: init-config
-        image: "gcr.io/datadoghq/agent:7.63.0"
-        imagePullPolicy: IfNotPresent
-        command: ["bash", "-c"]
-        args:
-          - for script in $(find /etc/cont-init.d/ -type f -name '*.sh' | sort) ; do bash $script ; done
-        volumeMounts:
-          - name: config
-            mountPath: /etc/datadog-agent
-            readOnly: false # Need RW for writing datadog.yaml config file
-        resources:
-          {}
+        - name: init-volume
+          image: "gcr.io/datadoghq/agent:7.63.0"
```

Comment:
for all this linter change. It would be good to sync with agent-onboarding team to know if we can commit in the repo the linter config so we can all use the same config.